pkg/domain/infra/tunnel/images.go

```diff
@@ -93,3 +94,20 @@ func (ir *ImageEngine) Pull(ctx context.Context, rawImage string, options entiti
 	}
 	return &entities.ImagePullReport{Images: pulledImages}, nil
 }
+
+func (ir *ImageEngine) Import(ctx context.Context, opts entities.ImageImportOptions) (*entities.ImageImportReport, error) {
+	var (
+		err       error
+		sourceURL *string
+		f         *os.File
+	)
+	if opts.SourceIsURL {
+		sourceURL = &opts.Source
+	} else {
+		f, err = os.Open(opts.Source)
```

Comment:
This is importing off the server, not the client.  Won't the user expect the file to be on their machine?

Reply:
We're still on the client here and are passing it to the bindings which eventually throw it over to the server.